Tools/Cluster.py

```diff
@@ -60,17 +64,24 @@ def Cluster_Ana(packdata, PFT_mask, ipft, var_pred_name, K, Nc):
         locations = np.array(A.index.to_list())
         cluster_dic["clus_%.2i_loc" % clus] = locations
         # 1.3 Randomly select Nc sites from each cluster
-        if len(locations) > Nc:
-            RandomS = random.sample(range(len(locations)), Nc)
-            SelectedID = locations[RandomS]
+        n = max(Nc, int(len(locations) * 0.2))
+        random.shuffle(locations)
```

Comment:
is it ensure that random use a pre-fixed seed value for reproducability?

Reply:
It is just for random sampling. After this line, we can simply get `n` random samples by `locations[:n]`